RSDKv5/RSDK/Dev/devfont_to_hpp.py

```diff
@@ -0,0 +1,74 @@
+# Requires the Pillow dependency
+import sys, os, math, itertools
+from PIL import Image
+from pathlib import Path
+
+WIDTH = 8
+HEIGHT = 8
+CHARCOUNT = 128
+COLCOUNT = 16
+
+IMGWIDTH = (COLCOUNT if CHARCOUNT >= COLCOUNT else CHARCOUNT) * WIDTH
+IMGHEIGHT = math.ceil(CHARCOUNT / COLCOUNT) * HEIGHT
+
+def encode_font(image_path):
+    print(f"Opening {image_path}...")
+
+    img = Image.open(image_path).convert("1")
+    iwidth, iheight = img.size
+    if iwidth != IMGWIDTH or iheight != IMGHEIGHT:
+        print(f"[ERROR] Image does not match expected size ({IMGWIDTH}x{IMGHEIGHT})")
+        return []
+
+    print("Encoding pixel data...")
+    data = list()
+
+    row = 0
+    col = 0
+    for char in range(CHARCOUNT):
+        px = char * (WIDTH * HEIGHT)
+        h = row * HEIGHT
+        for y in range(h, h + HEIGHT):
+            for x in range(WIDTH):
+                data.append("0x01" if img.getpixel((x + (col * WIDTH), y)) else "0x00")
+                px += 1
+        col += 1
+        if col >= COLCOUNT:
+            col = 0
+            row += 1
+
+    return data
+
+def main():
+    if len(sys.argv) == 1:
+        print("[ERROR] No input file given")
+    else:
```

Comment:
Return early to reduce indentation for the nominal path.
```suggestion
    if len(sys.argv) == 1:
        print("[ERROR] No input file given")
        return
```